apps/versions/urls.py

```diff
@@ -6,8 +6,10 @@
 
 urlpatterns = patterns(
     '',
-    url('^$', views.version_list, name='addons.versions'),
-    url('^format:rss$', VersionsRss(), name='addons.versions.rss'),
+    url('^(?:status:(?P<status>[^/]+))?$',
```

Comment:
By the way, if we stick with the "convoluted" regexps, then I think we need tests (that use `django.core.urlresolvers.resolve` to take the URL we want, and make sure it resolves to the views (and parameters) we expect. That would have caught the issues with the urls "not working".